src/renderers/materialized/index.ts

```diff
@@ -2,83 +2,84 @@ import { JsonForms } from '../../core';
 
 export * from './materialized.boolean.control';
 export * from './materialized.date.control';
+export * from './materialized.datetime.control';
 export * from './materialized.enum.control';
 export * from './materialized.integer.control';
 export * from './materialized.number.control';
 export * from './materialized.text.control';
 export * from './materialized.textarea.control';
 
 export const materialize = () => {
-  JsonForms.stylingRegistry.registerMany([
```

Comment:
Not a big issue but did you check tslint control? I think lint configuration is different in the project.